src/format_changeset.rs

```diff
@@ -0,0 +1,70 @@
+use difference::{Difference, Changeset};
+use std::fmt;
+use ansi_term::Colour::{Red, Green};
+use ansi_term::Style;
+
+macro_rules! paint {
+    ($f: ident, $colour: ident, $fmt: expr, $($args:tt)*) => (
+        write!($f, "{}", $colour.paint(format!($fmt, $($args)*)))
+    )
+}
+
+
+// Adapted from:
+// https://github.com/johannhof/difference.rs/blob/c5749ad7d82aa3d480c15cb61af9f6baa08f116f/examples/github-style.rs
+// Credits johannhof (MIT License)
```

Comment:
I've been told that we may put the entire file in the MIT-only license and state so in the LICENSE file. Is it something that's OK for you?